### PR TITLE
streamlit-navigation-bar 3.3.0 :snowflake:

### PR DESCRIPTION
**Pull request by psteyer**
streamlit-navigation-bar 3.3.0 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5228]
- dev_url:        https://github.com/gabrieltempass/streamlit-navigation-bar/tree/3.3.0
- conda_forge:    None
- pypi:           https://pypi.org/project/streamlit-navigation-bar/3.3.0
- pypi inspector: https://inspector.pypi.io/project/streamlit-navigation-bar/3.3.0/
- Related PR (dependencies)
    - https://github.com/AnacondaRecipes/st-theme-feedstock/pull/1
    - https://github.com/AnacondaRecipes/streamlit-feedstock/pull/11

### Explanation of changes:

- recipe creation and build for 3.3.0

[PKG-5228]: https://anaconda.atlassian.net/browse/PKG-5228?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by boldorider4 — APPROVED** [review]

**Comment by ifitchet** [comment]
`README.md` seems a bit short

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/streamlit-navigation-bar-{{ version }}.tar.gz
+  sha256: f458c3f9bbb72a3fa07a28c1db98eceabdd49b19f71fafb44dc36042e0d4b4c4
+
+build:
+  entry_points:
+    - streamlit-navigation-bar = streamlit_navigation_bar:print_version
+  skip: true  # [s390x or py<=38]
```
Comment (ifitchet) [inline recipe/meta.yaml]: `setup.py` says `python_requires=">=3.8"`:

```suggestion
  skip: true  # [s390x or py<38]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  home: https://github.com/gabrieltempass/streamlit-navigation-bar
+  summary: A component that allows you to place a navigation bar in your Streamlit app.
+  description: |
+    The navbar was built to be simple to use, look great out of the box, apply custom styles
+    integrate with streamlits UI, have a well-written documentation. It has some cool functionalities, 
+    like displaying an optional logo and external URLs. It also matches the active theme by default.
+  license: MIT
+  license_file: LICENSE
+  license_family: MIT
+  doc_url: https://github.com/gabrieltempass/streamlit-navigation-bar/wiki
```
Comment (ifitchet) [inline recipe/meta.yaml]: `setup.py` says:

```
"Documentation": "https://github.com/gabrieltempass/streamlit-navigation-bar/wiki/API-reference"
```

**Review by ifitchet — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (psteyer) [reply recipe/meta.yaml]: It seems like removing the `=` breaks the build.

**Review by Jrice1317 — APPROVED** [review]
LGTM

**Review by ifitchet — APPROVED** [review]
Nice README 😉

**Pull request merged**